framework/src/postprocessors/Receiver.C

```diff
@@ -29,18 +29,22 @@ Receiver::validParams()
 Receiver::Receiver(const InputParameters & params)
   : GeneralPostprocessor(params),
     _initialize_old(getParam<bool>("initialize_old")),
-    _my_value(getPostprocessorValueByName(name())) // use FEProblem to avoid cyclic dependency
+    _my_value(_fe_problem.getReporterData().getReporterValue<PostprocessorValue>(
```

Comment:
Why this change, seems like it should use the interface if it can.

Reply:
Yeah, I think you're right. Does the remaining stuff make sense, though? Needing write access to the data in order to request the older values. I'm trying to slowly wean off the use of the FEProblemBase calls (FEProblemBase::{getUserObject, getReporterValue, etc})